packages/flutter_tools/test/web.shard/sdk_web_configuration_test.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'package:dwds/dwds.dart';
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+import 'package:flutter_tools/src/artifacts.dart';
+import 'package:flutter_tools/src/isolated/sdk_web_configuration.dart';
+
+import '../src/common.dart';
+
+void main() {
+  FileSystem fileSystem;
+
+  group('Flutter SDK configuration for web', () {
+    final SdkConfigurationProvider provider = SdkWebConfigurationProvider(Artifacts.test(fileSystem: fileSystem));
```

Comment:
nit: I think you should move this to the `setUp()` step, as you are passing `fileSystem` once, and before it's initialized, and it is used in multiple tests.

Reply:
done, thanks!